templates/firefox-ios.xliff

```diff
@@ -3287,6 +3279,22 @@ Private Tab</source>
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.4" build-num="15F31d"/>
     </header>
     <body>
+      <trans-unit id="Bookmarks.EmptyState.Nested.Body.v135" xml:space="preserve">
+        <source>Add bookmarks as you browse so you can find your favorite sites later.</source>
+        <note>The body text for the placeholder screen shown when there are no saved bookmarks, located within a nested subfolder of the bookmarks panel within the libray modal</note>
```

Comment:
@MattLichtenstein
Not blocking, but there's a typo in all these comments that should be fixed (libray -> library).